lib/middleware.js

```diff
@@ -77,36 +77,51 @@ module.exports = function wrapper(context) {
             return goNext();
           }
         }
-        if (notFound) {
-          return resolve(goNext());
-        }
-        // server content
-        let content = context.fs.readFileSync(filename);
-        content = handleRangeHeaders(content, req, res);
-
-        let contentType = mime.getType(filename);
+        const defaultTransform = (localRequest, fs) => new Promise((localResolve) => {
+          if (notFound) {
+            return localResolve(null);
+          }
+          fs.readFile(localRequest.filename, (err, content) => {
```

Comment:
Is it a problem that the readFileSync was replaced by an async one?

Reply:
`fs` is a in memory filesystem so it doesn't matter